cpp/src/cider/exec/nextgen/function/CiderStringFunctions.cpp

```diff
@@ -140,48 +136,72 @@ const uint8_t ascii_char_lower_map[] = {
     224, 225, 226, 227, 228, 229, 230, 231, 232, 233, 234, 235, 236, 237, 238, 239,
     240, 241, 242, 243, 244, 245, 246, 247, 248, 249, 250, 251, 252, 253, 254, 255};
 
+ALWAYS_INLINE void do_lower(char* out, const char* in, int str_len) {
+#pragma clang loop vectorize(enable)
```

Comment:
it's compiler hints. and it's nice to have. see velox impl. https://github.com/facebookincubator/velox/blob/main/velox/functions/lib/string/StringCore.h#L91

Reply:
Will it be any difference or benefit if we add this line?

Reply:
No.

Reply:
So, remove this line and below.

Reply:
No. I will not remove this hints. I add this because I want compiler to generate vectorized instructions for this function no matter what LLVM default setting is.

Reply:
The point is you said there won't be any difference or benefit if we add this line, so why not remove them?

Ps. The result of your test proves the performance will be even worse if loop vectorized. (without so called ramp up)

Reply:
Like I said, this is compiler hint. this may be useful someday in the future.
you mean O1 optimization? that's due to cache, or something else.  if you exchange two code blocks, the result is opposite. Please run it.